### ISSUE TITLE
tf.contrib.opt.ScipyOptimizerInterface error

### ISSUE THREAD

**Issue by ciprianmihai94** [tensorflow/tensorflow#25279]
Hello.

I tried to use tf.contrib.opt.ScipyOptimizerInterface with an example code from your documentation.
`vector = tf.Variable([7., 7.], 'vector')`
`loss = tf.reduce_sum(tf.square(vector))`
`optimizer = tf.contrib.opt.ScipyOptimizerInterface(loss, options={'maxiter': 100})`
`with tf.Session() as session:`
`---->optimizer.minimize(session)`.

I am getting the following error: `Tensors in list passed to 'values' of 'ConcatV2' Op have types [int32, float32, float32, float32, float32, float32, float32, float32, float32, float32, int32, float32, float32, float32, float32, int32, int32, float32, float32, float32, float32] that don't all match.`

OS Platform and Distribution: Ubuntu 18.04.1
TensorFlow installed from: source
TensorFlow version: 1.12.0
Python version: 3.6
Bazel version: 0.21.0
GCC version: 7.3.0
CUDA and GPU: no running on GPU, because I don't have GPU.

Thank you!

**Comment by facaiy** [comment]
ScipyOptimizer is moving from contrib to tensorflow/addons in #3 . I'm afraid that it would be a while, at least, before we could look into the issue. cc @seanpmorgan

**Comment by seanpmorgan** [comment]
@facaiy is correct it will take some time before we're in a position to accept questions for addons ExternalOptimizers. As part of the moving and upgrading code from contrib to addons, we also need to step through the code and make sure we understand exactly how it works. 

In the mean time, I'll try to some troubleshooting -- but can not give any concrete time estimates.

**Comment by rthadur** [comment]
Contrib has been depreciated here in Tensorflow repo and moved to tensorflow/addons , please reopen the issue in  https://github.com/tensorflow/addons/issues if this exists in latest version. Thank you
